app/assets/stylesheets/components/_outcome-card.scss

```diff
@@ -0,0 +1,93 @@
+$outcome-card-height: 200px;
+
+.outcome-cards,
+.outcome-card {
+  @include clearfix;
+}
+
+.outcome-card {
+  background-color: $white;
+  color: $base-font-color;
+  display: block;
+  margin-bottom: $base-spacing;
+  padding: $base-spacing;
+  text-align: center;
+  white-space: normal;
+
+  @include media($medium-screen-up) {
+    @include span-columns(6 of 12);
+    @include omega(2n);
+    height: $outcome-card-height;
+    text-align: initial;
+  }
+}
+
+.outcome-card-unmatched {
+  border: $base-border;
+  transition: $base-transition;
+
+  &:focus,
+  &:hover {
+    border-color: $mit-red;
+    color: inherit;
+    transform: translate3d(0, -$smallest-spacing, 0);
+
+    .outcome-card-action {
+      transform: scale(1.0625);
+      transition: $base-transition;
+    }
+  }
+}
+
+.outcome-card-matched {
+  background-color: $lighter-gray;
+  cursor: default;
+}
+
+.outcome-card-copy {
+  @include media($medium-screen-up) {
+    float: left;
+    margin-right: $base-spacing;
+    width: 65%;
+  }
+
+  .outcome-card-matched & {
+    color: $medium-gray;
+    text-decoration: line-through;
+  }
+}
+
+.outcome-card-action {
+  color: $mit-red;
+  font-size: $smallest-font-size;
+  text-align: center;
+
+  @include media($medium-screen-up) {
+    float: right;
+    margin-top: $outcome-card-height / 4;
```

Comment:
This looks centered. Is the math here exact though?

If you wanted to be exact, you could take the card height and subtract the action height and then split the difference.